src/ECER.Resources.Accounts/PortalInvitations/PortalInvitationRepository.cs

```diff
@@ -0,0 +1,27 @@
+﻿using AutoMapper;
+using ECER.Utilities.DataverseSdk.Model;
+using Microsoft.Xrm.Sdk.Client;
+
+namespace ECER.Resources.Accounts.PortalInvitations;
+
+internal class PortalInvitationRepository : IPortalInvitationRepository
+{
+  private readonly EcerContext context;
+  private readonly IMapper mapper;
+
+  public PortalInvitationRepository(EcerContext context, IMapper mapper)
+  {
+    this.context = context;
+    this.mapper = mapper;
+  }
+
+  public async Task<IEnumerable<PortalInvitation>> Query(PortalInvitationQuery query)
+  {
+    await Task.CompletedTask;
+    var portalInvitations = context.ecer_PortalInvitationSet.Where(pi => pi.ecer_PortalInvitationId == query.portalInvitationId);
+
+    context.LoadProperties(portalInvitations, ecer_PortalInvitation.Fields.ecer_portalinvitation_ApplicantId);
```

Comment:
do you need the applicant or the related entity for that invite?

Reply:
I need applicant as well, to show their data in the page

Reply:
You need to get the context of this invite, it would be the application; then the manager should query the application, then query the applicant. You don't want to cross the service boundary here - invites are application related, not applicant.